crates/reth/evm/src/precompiles/bridge.rs

```diff
@@ -59,7 +54,11 @@ impl<DB: Database> ContextStatefulPrecompile<DB> for BridgeoutPrecompile {
         })?;
 
         // Log the bridge withdrawal intent
-        let evt = WithdrawalIntentEvent { amount, dest_pk };
+        let evt = WithdrawalIntentEvent {
+            amount,
+            // FIXME: How to make this work with a BOSD Descriptor.
+            destination,
```

Comment:
I would still probably pass it around as plain bytes instead of the encoded representation.  Once we've validated it we don't have to introspect it again until the operator gets it.  (Although we'd probably validate it once more when we have a cleaner break between the EVM EE and the Strata chain, still keeping it as bytes though.)